### ISSUE TITLE
Wallpaper download fails on HCI Box deployment (http/404)

### ISSUE THREAD

**Issue by ulkeba** [microsoft/azure_arc#2487]
**Is your issue related to a Jumpstart scenario, ArcBox, HCIBox, or Agora?**
[https://azurearcjumpstart.io/azure_jumpstart_hcibox/cloud_deployment](https://azurearcjumpstart.io/azure_jumpstart_hcibox/cloud_deployment)

**Describe the issue or the bug**
Inspecting the content of `C:\HCIBox\Logs\Bootstrap.log`, I see the following error:

```
PS>TerminatingError(Invoke-WebRequest): "404: Not Found"
Invoke-WebRequest : 404: Not Found
At C:\Packages\Plugins\Microsoft.Compute.CustomScriptExtension\1.10.15\Downloads\0\Bootstrap.ps1:99 char:1
+ Invoke-WebRequest "https://raw.githubusercontent.com/microsoft/azure_ ...
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidOperation: (System.Net.HttpWebRequest:HttpWebRequest) [Invoke-WebRequest], 
WebException
    + FullyQualifiedErrorId : WebCmdletWebResponseException,Microsoft.PowerShell.Commands.InvokeWebRequestCommand
Invoke-WebRequest : 404: Not Found
At C:\Packages\Plugins\Microsoft.Compute.CustomScriptExtension\1.10.15\Downloads\0\Bootstrap.ps1:99 char:1
+ Invoke-WebRequest "https://raw.githubusercontent.com/microsoft/azure_ ...
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidOperation: (System.Net.HttpWebRequest:HttpWebRequest) [Invoke-WebRequest], WebExc
   eption
    + FullyQualifiedErrorId : WebCmdletWebResponseException,Microsoft.PowerShell.Commands.InvokeWebRequestCommand
```

Line 99 of `Bootstrap.ps1` is:
```
Invoke-WebRequest "https://raw.githubusercontent.com/microsoft/azure_arc/main/img/hcibox_wallpaper.png" -OutFile $HCIPath\wallpaper.png
```

And indeed, no resource seems to be available at [https://raw.githubusercontent.com/microsoft/azure_arc/main/img/hcibox_wallpaper.png](https://raw.githubusercontent.com/microsoft/azure_arc/main/img/hcibox_wallpaper.png) (I get a http/404 in my browser, too).

**Comment by dkirby-ms** [comment]
Wondering if you deployed this last week as this issue was addressed in #2478

**Comment by ulkeba** [comment]
That might have been the case. I redeployed (for other reasons) and couldn't see the same behavior during the redeployment. So I think that request can be closed.

Sorry for the noise.
